gossip/blockproc/evmmodule/evm.go

```diff
@@ -29,6 +29,10 @@ func (p *EVMModule) Start(block iblockproc.BlockCtx, statedb *state.StateDB, rea
 	if block.Idx != 0 {
 		prevBlockHash = reader.GetHeader(common.Hash{}, uint64(block.Idx-1)).Hash
 	}
+
+	// Start block (Carmen specific)
```

Comment:
nit: the `(Carmen specific)` part should not be of concern for this function. Also below.